docs/source/models.rst

```diff
@@ -95,780 +95,36 @@ These files can then be used to reinstall dependencies using ``conda`` or ``virt
-
-How To Log Model With Json Object
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-For models accepting python dictionary inputs instead of pandas DataFrame, we support saving the example
-directly as it is. To enable this, ``example_no_conversion`` should be set to ``True`` when logging the model. 
-This feature is only supported for langchain, openai, pyfunc and transformers flavors, where saving the example
-directly is useful for inference and model serving.
-By default, ``example_no_conversion`` is set to ``False`` for backwards compatibility.
-
-The following example demonstrates how you can log a json object input example with your model:
-
-.. code-block:: python
+.. toctree::
+    :maxdepth: 1
+    :hidden:
 
-    input_example = {
-        "messages": [
-            {"role": "system", "content": "You are a helpful assistant."},
-            {"role": "assistant", "content": "What would you like to ask?"},
-            {"role": "user", "content": "Who owns MLflow?"},
-        ]
-    }
-    mlflow.langchain.log_model(..., input_example=input_example, example_no_conversion=True)
+    model/signatures
 
-How To Log Model With Example Containing Params
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-For models that require additional parameters during inference, you can include an input_example 
-containing params when saving or logging the model. To achieve this, the sample input should be 
-provided as a ``tuple``. The first element of the tuple is the input data example, and the
-second element is a ``dict`` of params. A comprehensive list of valid params is documented in
-:ref:`Model Inference Params <inference-params>` section.
+In MLflow, understanding the intricacies of model signatures and input examples is crucial for effective model management and deployment. 
+A model signature in MLflow describes the expected input and output formats for a model, ensuring consistency and correctness in model usage. 
+These signatures encompass various aspects, such as:
 
-* Python ``tuple``: (input_data, params)
+- **Model Signature**: Defines the schema for model inputs, outputs, and additional inference parameters, promoting a standardized interface for model interaction.
+- **Model Input Example**: Provides a concrete instance of valid model input, aiding in understanding and testing model requirements.
 
-The following example demonstrates how to log a model with an example containing params:
+Our documentation delves into several key areas:
 
-.. code-block:: python
+**Supported Signature Types**: We cover the different data types that are supported, such as tabular data for traditional machine learning models and tensors for deep learning models.
+**Signature Enforcement**: Discusses how MLflow enforces schema compliance, ensuring that the provided inputs match the model's expectations.
+**Logging Models with Signatures**: Guides on how to incorporate signatures when logging models, enhancing clarity and reliability in model operations.
```

Comment:
Wondering why `Logging Models with signatures` is not in a new line
<img width="1330" alt="image" src="https://github.com/mlflow/mlflow/assets/82044803/d3b38103-88de-4aa1-ab0e-00d2684f099c">

Reply:
I forgot the bullet points ;)